docs/docs/workflows/config.md

```diff
@@ -80,6 +80,50 @@ SNP-based attestation will be enabled as soon as a fix is verified.
 
 Fill the desired VM type into the **instanceType** fields in the `constellation-conf.yml` file.
 
+## Creating additional node groups
+
+By default, Constellation will create the node groups `control_plane_default` and `worker_default` for control-plane nodes and workers respectively.
+If you require additional control-plane or worker groups with different instance types, zone placements or disk sizes, you can add additional node groups to the `constellation-conf.yml` file.
+Each node group can be scaled individually.
```

Comment:
You already updated the "Scale your cluster" workflow some time ago. Is it still correct for the final implementation?

Reply:
Yes. The autoscaling docs already mention node groups and explain how users can choose a node group / scaling group to configure the autoscaling settings.